docs/sources/whatsnew/whats-new-in-v9-4.md

```diff
@@ -65,3 +65,9 @@ have a unified configuration style among all authentication providers. Instead o
 ## Loki datasource query validation
 
 We added support to validate queries and visually display errors as a query is being written, without having to execute it to receive this feedback. This feature supports single and multi-line queries, with and without variables.
+
+# Loki logs sample in Explore
+
+For Loki metric queries in Explore, you are now able to see the samples of log lines that contributed to the visualized metrics. To see the samples, simply click on collapsed Logs sample panel under your graph or table panel. If you would like to interact with your log lines or change the log query, you can click on Open logs in split view button and log query will be executed in the split view.
+
+{{< figure src="/media/docs/grafana/logs-sample-whats-new-9-4.png" max-width="750px" caption="Logs sample in Explore" >}}
```

Comment:
@robbymilo I just wanted to confirm that this the correct path we should use for newly uploaded images.

Reply:
That is correct: https://grafana.com/media/docs/grafana/logs-sample-whats-new-9-4.png